user/models.py

```diff
@@ -0,0 +1,25 @@
+"""
+User application model
+
+"""
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.urls import reverse
+
+
+class UserModel(AbstractUser):
+    genders = (
+        ("m", "Men"),
+        ("f", "Women")
+
+    )
+    sex = models.CharField(max_length=1, choices=genders, default="")
```

Comment:
Ось таке імʼя для цього поля буде більш правильним
```suggestion
    gender = models.CharField(max_length=1, choices=genders, default="")
```